src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -284,6 +285,17 @@ protected virtual void Initialize()
                 }
 
                 var bindingProviders = LoadBindingProviders(ScriptConfig, hostConfig, TraceWriter);
+
+
+                { // $$$
+                    var awesome = new GeneralScriptBindingProvider(ScriptConfig.HostConfig, hostConfig, TraceWriter);
+                    awesome.AddAssembly(typeof(ServiceBus.EventHubConfiguration).Assembly).Wait();
```

Comment:
This is key: demonstrate how we're adding an extension just by pointing to an assembly. Everything else is self-describing.